docker/nginx/loris.nginx.conf

```diff
@@ -42,8 +42,8 @@ http {
     # For example V0002007.jpg is actually stored in V0002000/V0002007.jpg in the s3 bucket.
     # Therefore every request for V0002007.jpg needs to be routed to s3:V0002000/V0002007.jpg
     #
-    location ~ "^/image/V[0-9]{4}.*" {
-      rewrite "^/image/(?<filename>(?<shard>V[0-9]{4}).*)" "/s3:${shard}000/${filename}" break;
+    location ~ "^/image/[A-Z][0-9]{4}.*" {
```

Comment:
I think there can be more than one letter at the beginning of a miro id. There is a similar regex [here](https://github.com/wellcometrust/platform/blob/42700ed7a65d1eade1bb95d9d92371b6ec615076/miro_preprocessor/miro_copy_s3_asset/src/miro_copy_s3_asset.py#L70). You might want to nick that

Reply:
👍